src/lib/clients/api.ts

```diff
@@ -0,0 +1,9 @@
+import createClient from 'openapi-fetch'
+import type { paths as InfraPaths } from '@/types/infra-api'
+
+export const infra = createClient<InfraPaths>({
+  baseUrl: process.env.INFRA_API_URL,
+  headers: {
+    'Content-Type': 'application/json',
+  },
```

Comment:
include default headers:
https://github.com/e2b-dev/E2B/blob/main/packages/js-sdk/src/api/metadata.ts#L44

Reply:
i think we'll need to include at least browser and platform headers for our analytics